docs/important/relevant_functionalities_de.md

```diff
@@ -0,0 +1,1250 @@
+
+Beispiele:
+- Reparatur korrupter Daten in einer Datenbank
+- Applikationsfehler
+- ungeplanter Stopp oder Restart von Diensten oder Applikationen
+- akute Konnektivitätsfehler
+    - z. B. Postgresql-Dogu ist auch nach mehreren Minuten nicht erreichbar
+- _wiederholte_, fehlgeschlagene Login-Versuche
+- akut drohender oder akut bestehender Mangel an Ressourcen
+- Meldungen mit `FATAL`- oder `SEVERE`-Log-Level
+
+##### `WARN`
+
+Log-Level analog zu `WARN` finden sich auch häufig auf Produktionssystemen.
+
+Logausgaben dieses Levels sind sehr ähnlich zu denen von `ERROR`. Im Unterschied zu diesen enthalten sie jedoch Ausgaben, die auf bevorstehende Fehler hinweisen.
+
+Beispiele:
+- fehlerhafte oder optimierbare Konfiguration wurde erkannt
+- Aktualisierung von Applikation oder Bibliotheken werden vorgeschlagen
+- mittelfristig drohender Mangel an Ressourcen
+- singuläre Konnektivitätsfehler, die zeitlich begrenzt sind
+    - z. B. Postgresql-Dogu ist beim ersten Versuch nicht erreichbar, doch nach wenigen Minuten schon
+- _einzelne_, fehlgeschlagene Login-Versuche
+- geplanter Start und Stopp von Diensten oder Applikationen
+
+##### `INFO`
+
+Ausgaben, die in besonderen, aber unkritischen Situationen von Interesse sein könnten. Ein _Catch-All_ für all die Ausgaben, die nicht wichtig genug für `WARN` sind, aber noch nicht tief in die Details der Anwendung einsteigen.
+
+Beispiele:
+- ein Subsystem wurde vollständig geladen
+- Fehlerloses Ergebnis von Hintergrundjobs
+- Erfolgreicher Start/Stop einer Komponente
+
+##### `DEBUG`
+
+Ausgaben, die in kritischen Situationen von Interesse sein könnten und/oder zur Fehlerfindung dienen.
+
+Beispiele:
+- eine relevante Entscheidung für eine Konfiguration wurde gefällt
+- Meldungen mit `TRACE`-Log-Level
+
+#### Mapping je Dogu
+
+Diese vier abstrakten Log-Levels haben hin und wieder keine direkte Beziehung zu konkreten Log-Levels in den Tools oder Plugins im Dogu und müssen in der Dogu-Entwicklung manuell abgebildet werden. Der Grund dafür liegt darin, dass sich sowohl von den Tools als auch damit einhergehenden Plugins und Frameworks die Log-Levels zu stark in Quantität und Qualität unterscheiden.
+
+Die Abbildung eines abstrakten Log-Levels liegt damit in der Hand der Dogu-Entwicklung. Hier kann man beispielsweise entscheiden, dass man das `TRACE`-Log-Level aus der Software auf das `DEBUG`-Log-Level des Dogus mappen möchte.
+
+Diese Abbildung von abstrakten zu konkreten Log-Levels birgt den Vorteil der Flexibilität, z. B. wenn mehrere unterschiedliche Konfigurationsdateien mit unterschiedlichen Log-Leveln gefüllt werden müssen.
+
+#### Wertebereich und Default
+
+Zum Einstellen des Log-Levels wird je Dogu ein optionaler Key `/config/${dogu}/logging/root` eingeführt, der einen der folgenden abstrakten Log-Level-Werte annehmen kann.
+
+- `ERROR`
+- `WARN` (Defaultwert)
+- `INFO`
+- `DEBUG`
+
+Wenn kein Key `/config/${dogu}/logging/root` existiert, dann wird automatisch `WARN` verwendet.
```

Comment:
Maybe it is worth a shot to mention who uses the default value if the `dogu.json` is configured properly?
```suggestion
Wenn kein Key `/config/${dogu}/logging/root` existiert, dann verwendet `doguctl` automatisch den Wert `WARN`, wenn dieser ordnungsgemäß [als Default-Wert](#validierung-und-default-werte) konfiguriert wurde.
```